x-pack/plugins/secops/public/components/empty_page/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,10 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders correctly 1`] = `
+<Component
+  actionLabel="Do Something"
+  actionUrl="my/url/from/nowwhere"
+  message="My awesome message"
+  title="My Super Title"
+/>
+`;
```

Comment:
I haven't used jest snapshots (yet) so I am assuming this snapshot is meant to be here for tests with this PR.